src/common/config/metadata/metadata.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata
+
+import (
+	"sync"
+)
+
+var metaDataOnce sync.Once
+var metaDataInstance *CfgMetaData
+
+// Instance - Get Instance, make it singleton because there is only one copy of metadata in an env
+func Instance() *CfgMetaData {
```

Comment:
I notice value calls the Instance(). GetMetaData(), could we use Meta.GetByKey() or MetaIns.GetByKey() instead?

Same as MetaIns.GetAll()

Reply:
Renamed